Frontend/Pages/Projects/LatestOfstedJudgement/Index.cshtml.cs

```diff
@@ -31,24 +28,21 @@ public class Index : CommonPageModel
             _getInformationForProject = getInformationForProject;
             _projectsRepository = projectsRepository;
         }
-        
+
         public async Task<IActionResult> OnGetAsync(bool addOrEditAdditionalInformation = false)
         {
             var projectInformation = await _getInformationForProject.Execute(Urn);
             var academy = projectInformation.OutgoingAcademies.First(a => a.Ukprn == AcademyUkprn);
-            var latestOfstedJudgement = academy.LatestOfstedJudgement;
-            
+            LatestOfstedJudgement = academy.LatestOfstedJudgement;
+
             OutgoingAcademyUrn = projectInformation.Project.OutgoingAcademyUrn;
             ProjectReference = projectInformation.Project.Reference;
-            SchoolName = latestOfstedJudgement.SchoolName;
-            InspectionDate = latestOfstedJudgement.InspectionDate;
-            OverallEffectiveness = latestOfstedJudgement.OverallEffectiveness;
-            OfstedReport = latestOfstedJudgement.OfstedReport;
+
             AdditionalInformationViewModel = new AdditionalInformationViewModel
             {
-                AdditionalInformation = latestOfstedJudgement.AdditionalInformation,
+                AdditionalInformation = LatestOfstedJudgement.AdditionalInformation,
                 HintText =
-                    "If you add comments, they'll be included in the latest Ofsted judgement section of your project template.",
+                        "This information will populate into your HTB template under the school performance (Ofsted information) section.",
```

Comment:
This HintText is out of date. Are we waiting on Adi to tell us what to put here? I think you could just go back to what it already said on Transfers